packages/@okta/vuepress-site/docs/reference/password-hook/index.md

```diff
@@ -97,7 +101,11 @@ For example, to indicate that the supplied credentials should not be accepted as
 
 If the default action sent by Okta in the `action.credential` property of the request to your external service was `UNVERIFIED`, then the same result, of rejecting the user-supplied credentials, could also be accomplished by means of returning an empty response with HTTP status code `204 NO CONTENT`. This would cause Okta to proceed with the default action.
 
-## Sample JSON Payload of Request
+### Timeout behavior
```

Comment:
Similar comment to earlier one on User Import hook - can we make this an H2 so it's shown at the top level of explorer tree?